rust-runtime/aws-smithy-runtime/src/client/http.rs

```diff
@@ -6,8 +6,38 @@
 /// Interceptor for connection poisoning.
 pub mod connection_poisoning;
 
+#[deprecated = "Direct HTTP test utility support from `aws-smithy-runtime` crate is deprecated. Please use the `test-util` feature from `aws-smithy-http-client` instead"]
```

Comment:
Note deprecating re-exports is currently not supported (https://github.com/rust-lang/rust/issues/30827). To do so, we need to use type aliases, e.g.,
```
#[deprecated = "...")]
pub type ... = ...;
```
But I think `pub use` is intended here because some of the deprecated items are functions and modules, to which we cannot add `pub type`.

Reply:
Good point though since we are deprecating the module I would assume anything in that module including re-exports is deprecated 🤔

Reply:
Yeah exactly, it'd be nice if it were supported.